src/components/seo.js

```diff
@@ -75,6 +77,7 @@ SEO.propTypes = {
   lang: PropTypes.string,
   keywords: PropTypes.arrayOf(PropTypes.string),
   title: PropTypes.string,
+  description: (props, propName) => validateDescription(props[propName]),
```

Comment:
Why not:
- Use the validation function directly?
- Skip the `propName` argument since we already know the prop name here?
- Use `PropTypes.string` in this short function (or inside the validation function), and change the validation to check only the length of the string?

Reply:
1. the function itself is both large enough and offtopic enough that I wanted to extract it to a diferent file
2. that makes the function unnecessarily brittle, and prone to mistakes if the name changes (which coincidentally, it will, given your suggestion of changing to `seo_description`). The argument is there already, so might as well use it, since there's no extra cost
3. not sure what you mean with either of these suggestions. can you provide samples? I don't know how to use `PropTypes.string` outside of assigning it to a prop directly

Reply:
Of course, my bad, should have included the suggestion in the end. Here's what I meant with point 3:

```javascript
  description: (props, propName) => PropTypes.string.isRequired(props, propName) || validateDescription(props[propName]),
```

You can also use `PropTypes.string.isRequired` inside the validation function, and change it here to:

```javascript
  description: validateDescription,
```

(This is what I meant with point 1, not inlining the function, sorry about the confusion :pray:)

Regarding point 2, what I meant is that you can simply assume the name of the prop that you already use in this very line. If it changes, you already have to change the line, I don't see how it is any more brittle. Example:

```javascript
  description: ({ description }) => validateDescription(description),
```